apps/src/studio/Item.js

```diff
@@ -347,9 +427,15 @@ Item.prototype.display = function () {
 
 Item.prototype.getNextPosition = function () {
   var unit = Direction.getUnitVector(this.dir);
+  var speed = this.speed;
+  // TODO: Better concept of which actions actually move the actor
+  // Projected position should not be in front of you if you are not moving!
+  if (this.activity === "none" || this.activity === "watchActor") {
+    speed = 0;
```

Comment:
It's true, the unit vector for `Direction.NONE` is (0, 0) - and we happen to reset the direction to Direction.NONE every tick, which causes nonmoving items to reset.  Now, with a nonmoving item that does have a rotation, we can't depend on that behavior.